ui/src/views/home/list.tsx

```diff
@@ -14,7 +14,7 @@ export const ListSites = () => {
     <Flex gap={10} mt="40px" flexWrap="wrap" justifyContent="center">
       {new Array(data).fill(0).map((_, index) => {
         const id = data - index - 1;
-        return <SiteCard tokenId={id} />;
+        return <SiteCard key={id} tokenId={id} />;
```

Comment:
I think we need another key value to assign since setting the index value to a key is not a good practice. Maybe use uuid for this

Reply:
But, as it's the tokenId itself maybe we can keep it like this

A question. What's gonna happen if we burn an NFT? Cause right now with this logic we assume that there are no burned NFT's

Reply:
You'll know a burned token because the owner will be `address(0)` so we can discard those in the list for now. And for later we can talk about if we want to surface them and if so, how (filters, burned trash bin kinda thing, etc).

Reply:
If we are able to get the latest token Id from the contract, then would we be able to stop using the index as part of the id calculation? `id` would start at that number and then just be `id -1` on the next round.

Reply:
Yeah, the thing is that we need to use map to render the cards, and I think zoruka did this so it renders the cards ordered descending